angular-workspace/projects/ni/nimble-angular/unit/byte/tests/nimble-unit-byte.directive.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { Component, ElementRef, ViewChild } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { NimbleUnitByteModule } from '../nimble-unit-byte.module';
+import { NimbleUnitByteDirective, UnitByte } from '../nimble-unit-byte.directive';
+
+describe('Nimble byte unit', () => {
+    describe('module', () => {
+        beforeEach(() => {
+            TestBed.configureTestingModule({
+                imports: [NimbleUnitByteModule]
+            });
+        });
+
+        it('custom element is defined', () => {
+            expect(customElements.get('nimble-unit-byte')).not.toBeUndefined();
+        });
+    });
+
+    describe('with template values', () => {
```

Comment:
We normally have a `describe` block called something like _with no values in template_, but I don't see that in this file. Was that intentional?

Reply:
Thanks, that was an oversight. Added.